lend/src/components/Balance/index.tsx

```diff
@@ -174,13 +213,22 @@ export const Balance = () => {
   const { account } = useWeb3Wallet();
   const Logo = getAssetLogo("USDC");
   const { loading, accountBalances } = usePoolAccountBalances();
-
+  const { referralLoading, referralAccountSummary } =
+    useContext(ReferralContext);
+  const rbnReferralRewards = referralAccountSummary["totalReferralRewards"];
   const { loading: depositLoading, totalDeposits } = usePoolTotalDeposits();
   const { loading: poolDataLoading, data: poolDatas } = usePoolsData();
   const { aprs } = usePoolsApr();
   const yourBalance = accountBalances.totalBalance;
-  const rbnClaimableRewards = accountBalances.rbnClaimable;
+  const rbnPoolRewards = accountBalances.rbnEarned;
   const rbnDecimals = getAssetDecimals("RBN");
+  const totalRbnRewards = rbnPoolRewards.add(
```

Comment:
we probably need useMemo here to calculate this to account for dependencies

Reply:
Addressed in [da914ae](https://github.com/ribbon-finance/ribbon-frontend/pull/624/commits/da914ae3a10539aa9d119085508d60ebfdf41c86)